README.md

```diff
@@ -155,3 +155,51 @@ Note: The file format on the save/compare feature isn't well defined, do
 consider it experimental as it might change without warning. OTOH right now
 being just a yaml list of bug numbers makes it very easy to - if needed - modify
 it.
+
+### Build and release the snap
+
+Snaps like [ustriage](https://snapcraft.io/ustriage) can be managed via the
+[my snaps interface](https://snapcraft.io/ustriage/listing) on snapcraft.io
+via the snap [release management](https://snapcraft.io/docs/release-management).
+
+There are usually two reasons to build and release a new revision of ustriage.
+On one hand changes we landed in the repository. On the other hand we sometimes
+do no-change rebuilds to pick up updated dependencies. The latter are usually
+needed once we got an automated notification that one such dependency had
+security updates.
+
+#### Builds for Code changes
+
+The [ustriage snap](https://snapcraft.io/ustriage) is configured to be linked to
+the [repository](https://github.com/canonical/ubuntu-server-triage) and
+therefore, whenever we push to the
+[repository](https://github.com/canonical/ubuntu-server-triage) it will
+automatically trigger new builds that one can find in the
+[builds overview](https://snapcraft.io/ustriage/builds).
+
+#### No change rebuilds
+
+If a rebuild is needed without a code change one can manually hit
+_"Trigger new build"_ at the top of the
+[builds overview](https://snapcraft.io/ustriage/builds) page of the snap.
+
+#### Releasing new snap revisions
+
+No matter which of the two ways above was triggering the builds, they will
+automatically create new revisions which will go to the `latest/edge` channel
+as seen in the [releases](https://snapcraft.io/ustriage/releases) overview.
+
+From there these new revisions can be verified manually, manually because so far
+we only have superficial tox testing for some basics. Furthermore and most
+meaningful functions needs valid launchpad credentials.
```

Comment:
```suggestion
From there, these new revisions can be verified manually (manually because so far
we only have superficial tox testing for some basics). Furthermore and most
meaningful functions need valid launchpad credentials.
```

Reply:
Also I'm not sure what we're trying to express by this "Furthermore and most meaningful functions need valid launchpad credentials."